Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,23 +1,20 @@
 import React from 'react';
 import './App.css';
+import Calculator from './Calculator';
 
-const App = () => {
-  return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
-    </div>
-  );
-};
+class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      total: 0
+    };
```

Comment:
I see what you were trying to do here! Great job for trying and working ahead in the material!